tests/run-make/link-framework/rmake.rs

```diff
@@ -0,0 +1,38 @@
+// Check that linking to a framework actually makes it to the linker.
+
+//@ only-apple
+
+use run_make_support::{cmd, rustc};
+
+fn main() {
+    rustc().input("dep-link-framework.rs").run();
+    rustc().input("dep-link-weak-framework.rs").run();
+
+    rustc().input("empty.rs").run();
+    cmd("otool").arg("-L").arg("no-link").run_fail().assert_stdout_not_contains("CoreFoundation");
+
+    rustc().input("link-framework.rs").run();
+    let out = cmd("otool")
```

Comment:
```suggestion
    cmd("otool")
```
Same two times before.

Reply:
Oh good catch! I applied your suggestion but completely forgot about the variables. Fixed. :)